api/api.go

```diff
@@ -135,6 +135,7 @@ func New(cfg Config, ms shuttletracker.ModelService, msg shuttletracker.MessageS
 
 	r.Get("/", api.IndexHandler)
 	r.Get("/about", api.IndexHandler)
+	r.Get("/test", api.IndexHandler)
```

Comment:
This should be removed

Reply:
Resolved in most recent commit

Was going to be for HTML schedules endpoint, but will remove for now